types/ramda/index.d.ts

```diff
@@ -214,7 +326,7 @@ export function any<T>(fn: (a: T) => boolean): (list: readonly T[]) => boolean;
  * ```typescript
  * const isClub = R.propEq('suit', '♣');
  * const isSpade = R.propEq('suit', '♠');
- * const isBlackCard = R.anyPass([isClub, isSpade]);
+ * const isBlackCard = R.anyPass<Card>([isClub, isSpade]);
```

Comment:
Should we add `Card` type/interface ?

Reply:
yeah, nice catch, thanks!

Reply:
just realized it doesn't quite work right now. in the other PR i had changed the type parameters to something i *think* is a bit more ergonomic, but might as well have examples that actually work for now. also made `anyPass` and `allPass` consistent